src/scss/modules/list-pages.scss

```diff
@@ -15,18 +15,64 @@
 }
 
 .list-page__title {
+  align-items: center;
   border-bottom: 1px solid $black-90;
   font-size: 24px;
   margin-bottom: 16px;
   padding-bottom: 16px;
 }
 
+.list-page__owner {
+  align-items: center;
+  margin-right: 16px;
+}
+
 .list-page__title-icon {
   margin-right: 16px;
-  vertical-align: bottom;
 
   .icon {
     font-size: 20px;
+    vertical-align: middle;
+  }
+}
+
+.list-page__owner-name-label {
+  white-space: nowrap;
+}
+
+.list-page__owner-name-filter {
+  button {
+    align-items: center;
+    border: none;
+    color: color(light_blue);
+    display: flex;
+    font-size: 24px;
+    font-weight: normal;
+    line-height: normal;
+    margin-left: 6px;
+    overflow: visible;
+    padding: 0;
+
+    .owner-component__name {
+      max-width: 268px;
+      overflow: hidden;
+      text-overflow: ellipsis;
+      white-space: nowrap;
```

Comment:
This is the code that handles the ellipsis overflow for long clinician/team names.

With that `max-width` set, it would support a max of `21 characters`. If the name is greater than that length, the `...` ellipsis is shown.

Reply:
Good call, updated in the last commit push.